tests/lib/rules/id-length.js

```diff
@@ -143,6 +147,21 @@ ruleTester.run("id-length", rule, {
         {
             code: "var { a: [x]} = {};",
             parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError,
```

Comment:
Do you know why this results in two warnings? Seems like it should only warn on `x` (`a` is not actually declared in this example).

Reply:
@kaicataldo 
Yes!. It's because of the issue - #12827 
Currently, this rule has a bug related to destructuring. I'll work on that in another PR.

Reply:
Got it! Thanks for explaining this :) Sounds like a good plan to me.